src/cpp/rtps/builtin/liveliness/WLP.cpp

```diff
@@ -112,25 +112,34 @@ bool WLP::initWL(RTPSParticipantImpl* p)
     pub_liveliness_manager_ = new LivelinessManager(
                 [&](const GUID_t& guid,
                     const LivelinessQosPolicyKind& kind,
-                    const Duration_t& lease_duration) -> void
+                    const Duration_t& lease_duration,
```

Comment:
pub_liveliness_changed could be used directly (no need for a lambda)

Reply:
It was agreed with @richiprosima that we would do it this way instead of using a `std::bind()`